lib/ansiblelint/__main__.py

```diff
@@ -151,7 +47,8 @@ def main():
 
     # no args triggers auto-detection mode
     if len(args) == 0 and not (options.listrules or options.listtags):
-        args = get_playbooks_and_roles(options=options)
```

Comment:
@cans @ssbarnea did we break auto-detection here (by removing `get_playbooks_and_roles()`)? Or does it still work?